index.html

```diff
@@ -67,17 +67,174 @@ <h2>I'm a software developer</h2>
+          I can help you build a product , feature or website Look through some of
+          my work and experience! If you like what you see and have a project you
+          need coded, don't hestiate to contact me.
+        </p>
+        <button class="resume-btn" type="button">Get my Resume</button>
+      </div>
+    </section>
+    <section class="Portfolio-boxes">
+      <div class="portfolio-lang-box">
+        <div class="portfolio-lang">
+          <div class="portfolio-globe d-flex d-flex-col">
+            <div class="globe-img d-flex">
+              <img class="globe-image" src="images/Icon-globe.svg" alt="globe-img" />
+            </div>
+            <h3>Languages</h3>
+            <ul class="globe-ul d-flex">
+              <li class="globe-li d-flex">Javascript</li>
+              <li class="globe-li d-flex">Ruby</li>
+              <li class="globe-li d-flex">Html</li>
+              <li class="globe-li d-flex">css</li>
+            </ul>
+          </div>
+        </div>
+        <div class="portfolio-lang">
+          <div class="portfolio-globe d-flex d-flex-col">
+            <div class="globe-img d-flex">
+              <img class="globe-image" src="images/Icon-frameworks.svg" alt="globe-img" />
+            </div>
+            <h3>Frameworks</h3>
+            <ul class="globe-ul d-flex">
+              <li class="globe-li d-flex">Bootstrap</li>
+              <li class="globe-li d-flex">Ruby on rails</li>
+              <li class="globe-li d-flex">RSpec</li>
+              <li class="globe-li d-flex">Capybara</li>
+              <li class="globe-li d-flex">Selenium</li>
+            </ul>
+          </div>
+        </div>
+        <div class="portfolio-lang">
+          <div class="portfolio-globe d-flex d-flex-col">
+            <div class="globe-img d-flex">
+              <img class="globe-image" src="images/Icon-skill.svg" alt="" />
+            </div>
+            <h3>Skills</h3>
+            <ul class="globe-ul d-flex">
+              <li class="globe-li d-flex">Codekit</li>
+              <li class="globe-li d-flex">Github</li>
+              <li class="globe-li d-flex">Codepen</li>
+              <li class="globe-li d-flex">Gitlab</li>
+              <li class="globe-li d-flex">Terminal</li>
+            </ul>
+          </div>
+        </div>
+      </div>
     </section>
+    <form class="form" action="https://formspree.io/f/mzbyokvj" method="post">
+      <h1 class="formHeading">I'm always interested in hearing about new projects, so if you'd like to chat please get
+        in touch.</h1>
+      <input type="text" id="name" name="name" placeholder="Full name" class="nameText" minlength="1" maxlength="30"
+        required />
+      <input type="email" id="email" name="email" placeholder="Email address" class="emailText" minlength="1"
+        required />
+      <textarea name="text" minlength="1" maxlength="500">Enter Your text here</textarea>
+      <button class="click" type="submit">Get in touch</button>
```

Comment:
Please implement button interactions such as hover, active (when it's pressed), disabled, and enable (default state).